SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -228,11 +228,6 @@
     <Message Importance="low" Condition=" $(SonarQubeTestProject) == 'true' "
              Text="SonarQube: project will be treated as a test project : $(MSBuildProjectName)" />
 
-    <PropertyGroup >
-      <!-- Ensure SonarQubeExclude has a value -->
-      <SonarQubeExclude Condition=" $(SonarQubeExclude) == '' ">false</SonarQubeExclude>
-    </PropertyGroup>
-
```

Comment:
I made this the corresponding task parameters optional rather than required so if this property isn't set then the task will default it to false, rather than throwing an error.